src/polysolve/linear/AMGCL_cuda.cu

```diff
@@ -0,0 +1,173 @@
+#ifdef POLYSOLVE_WITH_AMGCL
+// #ifdef POLYSOLVE_WITH_CUDA

```

Comment:
i think we need this

Reply:
I commented it out because polysolve CMakeLists.txt does not have POLYSOLVE_WITH_CUDA flag, but polyfem has. Shall I add the flag in CMakeLists.txt or delete this line of code?